src/api/models/summary_sample_events.py

```
@@ -647,6 +648,9 @@ class BaseProjectSummarySampleEvent(models.Model):
 
     class Meta:
         abstract = True
+        indexes = [
+            GinIndex(name="records_gin", fields=["records"]),
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)